assets/config/prod.json

```diff
@@ -1,4 +1,5 @@
 {
     "baseUrl": "https://apistaginghr.excellencetechnologies.in",
-    "apiUrl" : "https://apistaginghr.excellencetechnologies.in/attendance/API_HR/api.php"
+    "apiUrl" : "https://apistaginghr.excellencetechnologies.in/attendance/API_HR/api.php",
+    "salUrl" : "https://apistaginghr.excellencetechnologies.in/attendance/sal_info/api.php"
```

Comment:
this is not correct wayt to manage this.

https://medium.com/@matt.goodson.business/separating-build-environment-configurations-in-flutter-with-firebase-doing-it-the-right-way-c72c3ad3621f

read this or similar articles

Reply:
it's fixed